### ISSUE TITLE
Incorrect values returned for getPrevDate

### ISSUE THREAD

**Issue by ilya-v-trofimov** [P4sca1/cron-schedule#313]
Hi,
When running this code:
```
let schedule = parseCronExpression('0 10 * * 3')
wedSched.getPrevDate(new Date('2024-05-01T09:00:00'))
```
expected result is: `2024-04-24T00:00:00.000Z`
actual result is: `2024-03-27T00:00:00.000Z`

Node: 20.12.2

**Comment by P4sca1** [comment]
Hey @ilya-v-trofimov 👋
Thank you for pointing out this issue. It should be fixed in version 5.0.0. It is a major release, as support for Node.js 16 has also been dropped. Other than that, the API is unchanged. If you still have issues, please reopen this issue.

**Comment by P4sca1** [comment]
Release for version 5.0.0 failed. Use version 5.0.1 instead.
